translator/translate/otel/processor/transformprocessor/testdata/config.yaml

```diff
@@ -0,0 +1,28 @@
+metric_statements:
+  - context: resource
+    statements:
+      - keep_keys(attributes, ["ClusterName", "Namespace", "NodeName"])
+  - context: metric
+    statements:
+      - set(unit, "Bytes") where name == "jvm.memory.heap.used"
```

Comment:
nit: organize these so they are in order.. all jvm ones followed by all tomcat etc